chain-tx-enclave-next/enclave-ra/ra-client/src/verifier.rs

```diff
@@ -294,6 +322,14 @@ impl ClientCertVerifier for EnclaveCertVerifier {
 
 #[derive(Debug, Error)]
 pub enum EnclaveCertVerifierError {
+    #[error("Unable to parse attestation report: {0}")]
+    AttestationReportParsingError(#[source] serde_json::Error),
```

Comment:
Its better to use `#[from]` instead of `#[source]`. From `thiserror` [documentation](https://docs.rs/thiserror/1.0.20/thiserror/#details):

> The `#[from]` attribute always implies that the same field is `#[source]`, so you don't ever need to specify both attributes.

Reply:
Generic Error WebpkiError and JsonError have already done the auto casting of `webpki::Error` and `serde::Error` Can we leave the attestation report related errors to use `#[source]?